erpnext/stock/doctype/packing_slip/packing_slip.py

```diff
@@ -84,11 +84,14 @@ def get_details_for_packing(self):
 			* No. of Cases of this packing slip
 		"""
 
-		# also pick custom fields from delivery note
 		rows = [d.item_code for d in self.get("items")]
 
-		custom_fields = ', '.join(['dni.`{0}`'.format(d.fieldname) for d in \
-			frappe.get_meta("Delivery Note Item").get_custom_fields()])
+		# also pick custom fields from delivery note
+		exclude_fieldtypes = ["Column Break", "Section Break"]
```

Comment:
You can use `no_value_fields` defined in frappe/model/__init__.py

https://github.com/frappe/frappe/blob/develop/frappe/model/__init__.py#L10